ios/FluentUI/Pill Button Bar/PillButtonBar.swift

```diff
@@ -88,6 +88,22 @@ open class PillButtonBar: UIScrollView {
         }
     }
 
+    @objc public func disableItem(_ item: PillButtonBarItem) {
+        guard let index = indexOfButtonWithItem(item) else {
+            return
+        }
+
+        buttons[index].isEnabled = false
+    }
```

Comment:
How would a client re-enable an item?

Reply:
Added enableItem APi's as well now.
Missed adding them earlier.

Reply:
Added enableItem API

Reply:
Added enable api's

---
In reply to: [459589110](https://github.com/microsoft/fluentui-apple/pull/133#discussion_r459589110) [](ancestors = 459589110)